application/controllers/admin/tokens.php

```diff
@@ -2499,7 +2499,7 @@ public function _handletokenform($iSurveyId, $subaction, $iTokenId = "", $ajax =
      * @param int $iSurveyId
      * @return void
      */
-    private function _newtokentable($iSurveyId)
+    private function _newtokentable(int $iSurveyId)
```

Comment:
Is the typhint needed?
Are we using this for other method signatures?

Reply:
And it's great (because it send 400 error)